packages/react-scripts/config/env.js

```diff
@@ -27,6 +27,8 @@ function getClientEnvironment(publicUrl) {
       'NODE_ENV': JSON.stringify(
         process.env.NODE_ENV || 'development'
       ),
+      // Allows the chalk library to work on the browser
+      'FORCE_COLOR': JSON.stringify(true),
```

Comment:
Possibly (babel does some really weird things about moving requires/imports iirc), but I think asking for an option (e.g. forceColor) to be added here would be more appropriate https://github.com/babel/babel/blob/master/packages/babel-code-frame/src/index.js#L102.